index.html

```diff
@@ -4,41 +4,61 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css"/>
+    <title>About myself</title>
+    <link rel="stylesheet" href="style.css">
 </head>
 <body>
     <header>
-        <a class="logo" href="#">My Logo</a>
-        
-            <nav class="menu_icon"><img src="images/Union.png"/></nav>
+        <section>
+            <div class="headline">
+                <main>
+                    <h1>
+                        About Myself
+                    </h1>
+                    
+                    <p>
+                        Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                    </p>
+                    <p class="connect">LET'S CONNECT</p>
+                 <ul class="connect_menu">
+                    <li><a href="#"><img src="images/git.png"/></a></li>
+                    <li><a href="#"><img src="images/linkin.png"/></a></li>
+                    <li><a href="#"><img src="images/fingers.svg"></a></li>
+                    <li><a href="#"><img src="images/twitter.png"></a></li>
+                    <li><a href="#"><img src="images/M.png"></a></li>
+                 </ul>
+
+                <div class="resume"><button type="button">Get my resume</button></div>
+
+                
+                            
+                </main>
                   
-    </header>
```

Comment:
-  [ ] For the about page you should use a `section` tag  as required to wrap your about section